cluster-provision/k8s/1.17/provision.sh

```diff
@@ -2,6 +2,13 @@
 
 set -ex
 
+# Configure IPv6 DHCP
+nmcli connection modify "System eth0" ipv6.method dhcp
```

Comment:
You can have all these flags in one command, or exporting the NIC name to a variable

Reply:
i think its better this way, lets see other opinions

Reply:
I agree with @ormergi , if we need these (see previous question), then something like this is more suited:
```
nmcli connection modify "System eth0" \
    ipv6.method dhcp \
    ipv6.may-fail no \
    ...
&& \
nmcli connection up "System eth0"
```
I do not think the `nmcli connection down` is needed, `up` should be enough.

Reply:
about last comment line
Quique suggested here a reload 

Threre is a nmcli connection reload I think it will do same thing.
which is preferred ? (up only or reload)